### ISSUE TITLE
Issue logging in. Requires web authentication 

### ISSUE THREAD

**Issue by squk** [leikoilja/ha-google-home#533]
## Describe the bug

Issue logging in. Requires web authentication 

## Version of the integration

If you are unsure about the version check the [github release here](https://github.com/leikoilja/ha-google-home/releases).

```text
v1.9.10
```

## Logs

```text
2022-05-20 19:45:32 DEBUG (MainThread) [glocaltokens.client] Initializing new GLocalAuthenticationTokens instance.
2022-05-20 19:45:32 DEBUG (MainThread) [glocaltokens.client] Set GLocalAuthenticationTokens client access_token, homegraph, access_token_date and homegraph_date to None
2022-05-20 19:45:32 DEBUG (MainThread) [glocaltokens.client] Set GLocalAuthenticationTokens client username to "f*******************", password to "b***************", master_token to "" and android_id to
2022-05-20 19:45:32 DEBUG (SyncWorker_6) [glocaltokens.client] There is no stored master_token, logging in using username and password
2022-05-20 19:45:32 DEBUG (SyncWorker_6) [glocaltokens.client] There is no stored android_id, generating a new one
2022-05-20 19:45:32 DEBUG (SyncWorker_6) [glocaltokens.client] Generating android id...
2022-05-20 19:45:32 DEBUG (SyncWorker_6) [glocaltokens.client] Generated android id: 1d01d98e0b07586e
2022-05-20 19:45:33 ERROR (SyncWorker_6) [glocaltokens.client] [!] Could not get master token.
2022-05-20 19:45:33 DEBUG (SyncWorker_6) [glocaltokens.client] Request response: {'Error': 'NeedsBrowser', 'Url': 'REDACTED', 'ErrorDetail': 'To access your account, you must sign in on the web. Touch Next to start browser sign-in.'}
```

**Comment by ArnyminerZ** [comment]
This issue is currently being addressed at leikoilja/glocaltokens#245

**Comment by leikoilja** [comment]
Fixed in `ha-google-home` v1.9.11 🚀
https://github.com/leikoilja/ha-google-home/releases/tag/v1.9.11

Please update to the latest version and verify it checks :)
